File: CPP/Program/Car.cpp
#pragma once
// Memasukan library yang digunakan
#include <iostream>

#include <string>

#include "Vehicle.cpp"


// Deklarasi namespace
using namespace std;

// Inheritance dengan file Vehicle
class Car : public Vehicle
{
private:
   // Atribut Privat
    string jumlahKursi;
    string jumlahPintu;
public:
     // Constructor
    Car()
    {

    }

    //Constructor dengan Parameter
    Car(string jumlahKursi, string jumlahPintu)
    {
        this->jumlahKursi = jumlahKursi;
        this->jumlahPintu = jumlahPintu;

    }

    // Setter dan Getter
    void setJumlahKursi(string jumlahKursi)
    {
        this->jumlahKursi = jumlahKursi;
    }

    void setJumlahPintu(string jumlahPintu)
    {
        this->jumlahPintu = jumlahPintu;
    }

    string getjumlahKursi()
    {
        return this->jumlahKursi;
    }

    string getJumlahPintu()
    {
        return this->jumlahPintu;
    }

    // Destructor
    ~Car()
    {

    }
};

// Car::Car(/* args */)
// {
// }

// Car::~Car()
// {
// }

File: Python/Program/Car.py
## Saya Muhammad Rifky Afandi dengan NIM 2202346 mengerjakan LP2 dalam mata kuliah 
## Desain Pemorograman Berbasis Objek untuk keberkahanNya maka saya tidak 
## melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin.

## Mengimpor file
from Vehicle import Vehicle

## Membuat class dan inheritance dengan Vehicle
class Car(Vehicle):
    ## Atribut privat
    jumlahKursi=""
    jumlahPintu=""
    
    ## Constructor
    ## Atribut Publik
    def __init__(self, jumlahKursi="", jumlahPintu=""):
        super().__init__()
        self.jumlahKursi = jumlahKursi
        self.jumlahPintu = jumlahPintu

    ## Getter dan Setter
    def setJumlahKursi(self, jumlahKursi):
        self.jumlahKursi = jumlahKursi

    def setJumlahPintu(self, jumlahPintu):
        self.jumlahPintu = jumlahPintu

    def getJumlahKursi(self):
        return self.jumlahKursi

    def getJumlahPintu(self):
        return self.jumlahPintu